build.sbt

```diff
@@ -43,7 +43,7 @@ lazy val smetrics = (project
     libraryDependencies ++= Seq(
       Cats.core,
       Cats.effect,
-      `cats-helper` % Test,
+      `cats-helper`,
```

Comment:
does `cats-helper` depend on `smetrics`?

Reply:
Definitely no, it would be cross-dependency then